README.md

```diff
@@ -177,33 +177,31 @@ class { 'splunk::enterprise':
 }
 ```
 
-Alternatively the the `splunk::enterprise::password::seed` class can be used independently of the Puppet Agent through a [Bolt Plan apply block](https://puppet.com/docs/bolt/latest/applying_manifest_blocks.html).
+Alternatively the `splunk::enterprise::password::seed` class can be used independently of the Puppet Agent through a [Bolt Plan apply block](https://puppet.com/docs/bolt/latest/applying_manifest_blocks.html).
 
 ### Upgrade splunk and splunkforwarder packages
 
 This module has the ability to install *and* upgrade the splunk and splunkforwarder packages. All you have to do is declare `package_ensure => 'latest'` when calling the `::splunk` or `::splunk::forwarder` classes.
 
-Upgrades from 7.0.X to >= 7.0.X are not tested.
-
 #### Upgrade Example
 
-The following code will install the 6.6.8 version of the splunk forwarder. Then
-comment out the 6.6.8 version and build values and uncomment the 7.1.2 version
+The following code will install the 9.1.0 version of the splunk forwarder. Then
+comment out the 9.1.0 version and build values and uncomment the 9.2.0.1 version
 and build values. Running puppet again will perform the following:
 
 1. splunk forwarder package is upgraded
     1. splunk service is stopped as part of the package upgrade process
-1. new license agreement is automatically accepted
+2. new license agreement is automatically accepted
     1. license agreement must be accepted or the splunk service will fail to start
-1. splunk service is started
+3. splunk service is started
 
 ```puppet
 # Tell the module to get packages directly from Splunk.
 class { 'splunk::params':
-  version  => '6.6.8',
-  build    => '6c27a8439c1e',
-  #version  => '7.1.2',
-  #build    => 'a0c72a66db66',
+  version  => '9.1.0',
+  build    => '1c86ca0bacc3',
+  #version  => '9.2.0.1',
```

Comment:
do these commented parameters have any additional value?

Reply:
Read the section above, they're here for demonstrating an upgrade 😄